openedx/frontend-app-course-authoring
# fix: export and import UI bugs

JIRA Ticket: [TNL-11052](https://2u-internal.atlassian.net/browse/TNL-11052)

> 1. Change learn more button to link
> 2. Main body text should be small  
> 3. Update alert modal component to include icon  
> 4. Fix header bottom margin  
> 5. Update icons to be `CheckCircle` when step completes for export/import  
> 6. Update table to take up full space of section
> 7. Update success date and time to be localized  
> 8. fix wrapping on sidebar
> 9. Move selected file message to top of import card  
> 10. Fix missing word studio in export and import sidebar messages

### Testing

Import
1. import a course to see the icons change from gears to `CheckCircle`
2. Check the success time is not utc and matchs expected time zone.
3. After import, the name of the imported file should appear above the dropzone area
4. Check that the learn more text in the side bar is a link

Export

1. export a course to see the icons change from gears to `CheckCircle`
2. Check the success time is not utc and matchs expected time zone.
3. Check that the learn more text in the side bar is a link
